packages/qwik-city/buildtime/vite/image-jsx.ts

```diff
@@ -84,17 +134,25 @@ export function imagePlugin(userOpts?: QwikCityVitePluginOptions): PluginOption[
           const extension = path.extname(pathId).toLowerCase();
 
           if (supportedExtensions.includes(extension)) {
-            if (!code.includes('srcSet')) {
+            if (!code.includes('sources')) {
               this.error(`Image '${id}' could not be optimized to JSX`);
             }
             const index = code.indexOf('export default');
             return (
               code.slice(0, index) +
               `
   import { _jsxQ } from '@builder.io/qwik';
-  const PROPS = {srcSet, width, height};
+  const PROPS = {...img};
   export default function (props, key, _, dev) {
-    return _jsxQ('img', {...{decoding: 'async', loading: 'lazy'}, ...props}, PROPS, undefined, 3, key, dev);
+    if(!sources?.length) return null;
+    if(sources.length < 2){
+      // Single format directly returns the <img />
+      return _jsxQ('img', {...{decoding: 'async', loading: 'lazy'}, ...props}, sources[0], undefined, 3, key, dev);
```

Comment:
Since 1.8.1 we can use jsx inside qwik-city. This will also make it work with v2. So could you rename this file to .tsx and then just use `<img decoding="async"...`? There's also some merge conflicts so perhaps best to squash and rebase.